docs/ref/models/fields.txt

```diff
@@ -1638,6 +1638,31 @@ The possible values for :attr:`~ForeignKey.on_delete` are found in
     integrity, this will cause an :exc:`~django.db.IntegrityError` unless
     you manually add an SQL ``ON DELETE`` constraint to the database field.
 
+* .. attribute:: DB_CASCADE
+
+    .. versionadded:: 4.0
+
+    Django takes no direct action, behaves exactly like ``DO_NOTHING``,
+    but also sets ``ON DELETE CASCADE`` as a SQL constraint. The cascade
```

Comment:
bump to 4.1,  and "an" SQL (ess-queue-ell in Django docs convention). same directly below.